pkgs/by-name/ap/apko/package.nix

```diff
@@ -85,7 +85,7 @@ buildGoModule (finalAttrs: {
 
   meta = {
     homepage = "https://apko.dev/";
-    changelog = "https://github.com/chainguard-dev/apko/blob/main/NEWS.md";
+    changelog = "https://github.com/chainguard-dev/apko/releases/tag/v${finalAttrs.version}";
```

Comment:
You clearly changed this line _before_ running `nixpkgs-update`. So why didn't you commit it first?

Just like last time, our commit conventions ask for one commit for each logical unit.
The changelog URL didn't change as part of 0.29.2.